packages/models/src/openai/chat.ts

```diff
@@ -7,16 +7,24 @@ const OPENAI_CHAT_COMPLETIONS_API_URL = 'https://api.openai.com/v1/chat/completi
 export namespace OpenAIChatTypes {
   export type RequestOptions = SharedRequestOptions;
 
+  export type OpenAIFunction = {
```

Comment:
Nit but I'd name this without the OpenAI prefix given it's already namespaced under openai. IMO `Function`, `FunctionDefinition`, or `FunctionType` would be best.

Reply:
Ah, I was worried about overriding the keyword Function, even in the namespace.